### PR TITLE
Added forum class

### PR DESCRIPTION
**Pull request by Byambaa0325**
Added a forum class to create forum structure:
-stores article/post ids
-stores members
-stores owners

### CODE REVIEW

**Comment by Byambaa0325** [comment]
This is initial commit to making forum. More to come

File: src/main/java/com/google/codeu/data/Forum.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import java.util.List;
+import java.util.UUID;
+
+public class Forum {
+    private UUID id;
+    private String title;
+    private List<User> owners;
+    private List<User> members;
+    private List<String> keywords;
+    private List<UUID> articleIds;
```
Comment (tugushka) [inline src/main/java/com/google/codeu/data/Forum.java:12]: I think you've forgotten to add constructor/modifier to articleIds

File: src/main/java/com/google/codeu/data/Forum.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+
+    public Forum(UUID id, String title, List<User> owners, List<User> members, List<String> keywords, List<ForumSection> sections) {
```
Comment (tugushka) [inline src/main/java/com/google/codeu/data/Forum.java]: What does ForumSection do? Maybe you should add class member, at least.

Comment (tugushka) [reply src/main/java/com/google/codeu/data/Forum.java]: > add class member

to Forum

**Review by tugushka — COMMENTED** [review]
Why do we have a members and owners variable? Also add List<ForumSection>section to the class

**Comment by Byambaa0325** [comment]
Please nevermind the forum section I scrambled it during branching. I'll fix it in a sec.

**Comment by Byambaa0325** [comment]
For the owners and members variable, in future, we might want to have user groups or custom forums which users can create. For default forums, such as regions, members would include all users and owners would be admins.

**Review by tugushka — APPROVED** [review]

**Pull request merged**